traits/trait_set_object.py

```diff
@@ -338,179 +300,93 @@ def symmetric_difference_update(self, value):
             self.notify(removed, added)
```

Comment:
Comment on lines 298-299 (GitHub does not allow me comment directly on them).
Nit: If you replace these two lines with a call to `super().symmetric_difference_update` above the `if removed or added:`, that would also be fine and would be more consistent with the other changes, something like that:
```
        super().symmetric_difference_update(removed | added)
        if removed or added:
            self.notify(removed, added)
```

Reply:
Thanks!